mridefacer

```diff
@@ -76,6 +84,21 @@ Options:
   versions. In this case, the resulting images will be in FSL's
   standard radiological orientation.
 
+--apply-only
+  If this flag is set, the defacing is applied to the input images,
+  and stored into a new file (with _defaced suffix) instead of replacing
+  original one.
+
+--keep-orig
+  If this flag is set (in effect only if --apply), original image will be
+  stored with _orig suffix.
+
+--annex
+  If this flag is set, it will add all produced niftis to git annex, and
+  annotate original anatomical with metadata
+    distribution-restrictions=sensitive (datalad convention)
+  to make it easy to automate non-redistribution of these materials.
```

Comment:
I think this argument should take a value which by default is `'distribution-restrictions=sensitive'`, but would otherwise allow overwriting the default.

Reply:
is it doable with shell's getopt to make value "optional"?